it/uk/gov/hmrc/minorentityidentificationfrontend/stubs/StorageStub.scala

```diff
@@ -60,6 +60,13 @@ trait StorageStub extends WiremockMethods {
       status = status
     )
 
+  def stubStoreOverseasTaxIdentifiersCountry(journeyId: String, taxIdentifiersCountry: String)(status: Int): Unit =
+    when(method = PUT,
+      uri = s"/minor-entity-identification/journey/$journeyId/country", body = Json.toJson(taxIdentifiersCountry)
```

Comment:
This is obviously working, but other stubs for string data fields tend to use 
body = JsString(<identifier>).

Reply:
done